README.rst

```diff
@@ -70,6 +70,8 @@ This client supports the following Google Cloud Platform services at an Alpha_ q
 If you need support for other Google APIs, check out the
 `Google APIs Python Client library`_.
 
+.. _Alpha: https://cloud.google.com/terms/launch-stages
+.. _Beta: https://cloud.google.com/terms/launch-stages
```

Comment:
What's the point of having two things point to the same link? (Honest question, feels like it's not worthwhile, but maybe I'm wrong.)